spring-boot-modules/spring-boot-properties-3/src/main/java/com/baeldung/error/configuration/MainConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.error.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * <h2>This class is for Configuration.</h2>
+ * <p>
+ * Project Name: New folder
+ *
+ * @author Zahid Khan
+ * @version 4/1/2023
+ */
+@Configuration
+@ConfigurationProperties(prefix = "profile.application-properties")
```

Comment:
prefix value is different than the one mentioned in the draft: 
https://drafts.baeldung.com/?p=159983&preview=true#basic-configuration

Reply:
Updated the prefix and draft to keep in sync.